sphinxcontrib/inmanta/pygments_lexer.py

```diff
@@ -71,7 +71,7 @@ def process_id(self, match):
             ("[\"]{3}([\\n]|.)*?[\"]{3}", Comment.Multiline),  # t_begin_mls
             ("\n+", Whitespace),  # t_ANY_newline
             ('!=|==|>=|<=|<|>', Token.Operator),  # t_CMP_OP
-            ('[:[\]()=,.{}*]', Token.Operator),  # literals
+            ('[:[\]()=,.{}\?*]|(\+=)', Token.Operator),  # literals
```

Comment:
I assume it had trouble with `c ? x : y` as well. This should also be fixed with this change, nice.